src/webgpu/web_platform/canvas/getCurrentTexture.spec.ts

```diff
@@ -179,21 +210,7 @@ g.test('multiple_frames')
         prevTexture = currentTexture;
 
         if (frameCount++ < 5) {
-          // Which method will be used to begin a new "frame"?
-          switch (canvasType) {
-            case 'onscreen':
-              requestAnimationFrame(frameCheck);
```

Comment:
On the first iteration I think this will call getCurrentTexture twice in the same frame. The first frameCheck isn't scheduled by rAF, so the rAF called from it may be run at the end of the same frame.

Fix would I think be to initially call frameCheck from requestAnimationFrame on line L202/R219 below.